recipes/lhcbdirac-config/meta.yaml

```diff
@@ -0,0 +1,28 @@
+{% set name = "lhcbdirac-config" %}
+{% set version = "1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+build:
+  noarch: generic
+  number: 0
+
+requirements:
+  run:
+    - ca-policy-lcg
+
+test:
+  commands:
+    - test -f "${CONDA_PREFIX}/etc/dirac.cfg"
+    - echo "${X509_VOMS_DIR}"
+
+about:
+  home: https://lhcb-dirac.readthedocs.io/
+  license: None
```

Comment:
If there is no license then copyright is automatic. Can you ask open this upstream?

Reply:
This package (`lhcbdirac-config`) only contains a configuration file that is included in the feedstock directory. All of the code is in the `lhcbdirac` package which is GPL-3.0. I would argue there isn't anything to copyright as it is just contains 5 URLs:

```
DIRAC
{
  Setup = LHCb-Production
  Configuration
  {
    Servers = dips://lbvobox100.cern.ch:9135/Configuration/Server
    Servers += dips://lbvobox101.cern.ch:9135/Configuration/Server
    Servers += dips://lbvobox102.cern.ch:9135/Configuration/Server
    Servers += dips://lbvobox103.cern.ch:9135/Configuration/Server
  }
}
LocalInstallation
{
  Setup = LHCb-Production
  ConfigurationServer = dips://lhcb-conf-dirac.cern.ch:9135/Configuration/Server
}
```
The reason for a separate package is incase this ever changes, but that is unlikely. If you think is need to have a licence attached I'll ask the maintainers of this installation.

Reply:
The license is pretty clearly GPL v3, from what I can tell.

Reply:
There are [two recipes here](https://github.com/chrisburr/staged-recipes/tree/add-lhcbdirac/recipes):
* `lhcbdirac`: Clearly GPL-3.0 and the `meta.yaml` reflects this [here](https://github.com/conda-forge/staged-recipes/pull/8387/files#diff-04b7be3e94abf03f8ac73e49a4ea53c6R35)
* `lhcbdirac-config`: Contains only configuration files and currently marked with `license: None`

Reply:
Indeed but `lhcbdirac-config` is used with `lhcbdirac` and, based on my limited knowledge of GPL-3.0,  `lhcbdirac-config` must be GPL'ed too.

Reply:
I've spoken to the authors and they're happy for it to be GPL 3.